app/models/food.rb

```diff
@@ -0,0 +1,9 @@
+class Food < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods, dependent: :destroy
+  has_many :recipes, through: :recipe_foods
+
```

Comment:
- [OPTIONAL] The `Recipe` model doesn't belong to the `Food` model based on the schema diagram. They have a **_many-to-many_** relationship through the `Recipe_Food` model. I would advise that you establish the relationship between these models correctly. 👍 

#### Hint: You can read their [documentation](https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association:~:text=2.4%20The%20has_many%20%3Athrough%20Association) to understand how to go about it. 🥂